mmv1/products/parametermanager/Parameter.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'Parameter'
+description: |
+  A Parameter resource is a logical parameter.
+references:
+  guides:
+  api: 'https://cloud.google.com/secret-manager/parameter-manager/docs/reference/rest/v1/projects.locations.parameters'
+docs:
+base_url: 'projects/{{project}}/locations/global/parameters'
```

Comment:
Will parameters ever be location-based? Or are they strictly limited to global forever?

Reply:
The API is regional and global is considered as a region ( so base URL can be parameterised)
However the URL for global and regional API endpoints vary:
**Global**: parametermanager.googleapis.com
**Regional**: parametermanager.**{location}.rep.**googleapis.com

@slevenick Just because of this we will have to make keep regional and global resources. Can these be combined?

Reply:
Hmmmm, does the regional endpoint work for global? Like `parametermanager.**global.rep.**googleapis.com`?

If so, it should be easy to regionalize it now and allow global as a region.

Otherwise it should be possible, but may require some special casing. A lot of it depends on how you want to surface these resources to customers. Should they be separate resources in Terraform or not?

Reply:
Does the regional endpoint work for global? Like parametermanager.**global.rep.**googleapis.com?
 - It will not for global only parametermanager.googleapis.com will work

Regarding separate resources: Given that we don't have any change in logic, I don't have a strong reason why we should have separate resources apart from segregating the endpoints.